README.md

```diff
@@ -44,6 +44,9 @@ Get Involved: [Discuss and contribute on GitHub](https://github.com/yanolja/aren
    GOOGLE_CLOUD_PROJECT=<your project id> \
    CREDENTIALS_PATH=<your crednetials path> \
    MODELS_SECRET=<your secret> \
+   RATINGS_COLLECTION=<your collection> \
```

Comment:
Maybe it will be nicer to use `python-dotenv` to load these variables from env file.
Just a suggestion, you don't need to implement it here.

Reply:
I'll handle it in another PR. thank you!